core/src/main/scala/org/apache/spark/sql/TiExtensions.scala

```diff
@@ -68,7 +68,7 @@ object TiExtensions {
    * Catalog for tidb is necessary now.
    * @param sparkSession
    */
-  def validateCatalog(sparkSession: SparkSession): Unit = {
+  def validateCatalog(sparkSession: SparkSession): String = {
```

Comment:
the name should not be the `validateCatalog` for it can also return the catalog name